lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2843,34 +2865,36 @@ void PartitionCloner::handleSwitchEnum(SwitchEnumInst *inst) {
                    "from Swift to TensorFlow.");
 
   // Create the host->accelerator sends in the host code.
-  for (unsigned i = 0, e = inst->getNumCases(); i != e; ++i) {
-    SILBasicBlock *caseBB = inst->getCase(i).second;
+  unsigned caseId = 0;
+  for (auto *caseBB : inst->getSuccessorBlocks()) {
     SILBuilder BH(caseBB); // Builder for host.
 
     // The send is at the beginning of each caseBB.
     BH.setInsertionPoint(&caseBB->front());
     auto loc = caseBB->front().getLoc();
     // `caseId` must be of a type conforming to `AccelerableByTensorFlow`, so we
     // chose Int64 here.
```

Comment:
FYI, Int32 is the type that works on most devices - a lot of things don't support int64.

Reply:
Ack. changed to int32.